SPECS/isert-hwe/isert-hwe.spec

```
@@ -66,16 +66,17 @@
 
 Summary:	 %{_name}-hwe Driver
 Name:		 isert-hwe
-Version:	 24.10
-Release:	 23%{release_suffix}%{?dist}
+Version:	 25.07
+Release:	 1%{release_suffix}%{?dist}
 License:	 GPLv2
 Url:		 http://www.mellanox.com
 Group:		 System Environment/Base
-Source0:         https://linux.mellanox.com/public/repo/mlnx_ofed/24.10-0.7.0.0/SRPMS/isert-24.10.tgz#/isert-%{_version}.tgz
+# DOCA OFED feature sources come from the following repo MLNX_OFED_SRC tgz
+# https://linux.mellanox.com/public/repo/doca/3.1.0/SOURCES/mlnx_ofed/MLNX_OFED_SRC-25.07-0.9.7.0.tgz
+Source0:         %{_distro_sources_url}/isert-%{_version}.tgz
```

No need for the comment, you can just keep the URL in the `Source` tag:

```suggestion
Source0:         https://linux.mellanox.com/public/repo/doca/3.1.0/SOURCES/mlnx_ofed/MLNX_OFED_SRC-25.07-0.9.7.0.tgz#/isert-%{_version}.tgz
```